PodFile

```diff
@@ -1,16 +1,5 @@
 use_frameworks!
 
-def import_base_pods
-    pod 'FMDB', '= 2.6'
-    pod 'CocoaLumberjack', '~> 2.0'
-end
-
-def import_encrypted_base_pods
-    pod 'FMDB/SQLCipher', '= 2.6'
-    pod 'SQLCipher/fts', '~> 3.1.0'
-    pod 'CocoaLumberjack', '~> 2.0'
-end
-
 def import_test_pods
```

Comment:
Seems like this could be simplified to:
```ruby
def import_test_pods
    import_encrypted_test_pods
    pod 'MRDatabaseContentChecker'
end
```

Although the Podfile guide seems tot suggest using `abstract_target` e.g.

```ruby
abstract_target 'test_pods' do
    pod 'Specta'
    pod 'Expecta'
    pod 'OCMock'
    pod 'OHHTTPStubs'

    target 'CDTDatastoreTests' do
        pod 'MRDatabaseContentChecker'
        inherit! :complete
    end

    target 'CDTDatastoreEncryptionTests' do
        inherit! :complete
    end
end
```